README.md

```diff
@@ -1 +1,73 @@
-# hello-microverse
```

Comment:
Well done on using a custom README. It makes your repo look professional. You seem to have forgotten to customise it to the project. It is required to do so in order to have your project approved. Take a look at the comments below in the README on how you can make your it pop 😺 .

If you need more details on this topic, please have read a lesson [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).